src/main/scala/rps/controller/GameController.scala

```diff
@@ -0,0 +1,40 @@
+package rps.controller
+
+import rps.model.{GameSummary, Move, Result}
+import Move._, Result._
+import rps.model.error._
+import rps.service.{GameService}
+import rps.repository.{GameRepository}
+import GameRepository._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+import wiro.annotation._
+
+package GameController {
```

Comment:
This package seems redundant, I would just leave `GameApi` in the `controller` package (you can rename it `GameController` if you want.

In general, also, this syntax is not used in Scala. When using a subpackage just create a new suddirectory name `mysubpackage` and create files within it.